mypyc/build.py

```diff
@@ -513,7 +513,7 @@ def mypycify(
         cflags += [
             '-O{}'.format(opt_level), '-Werror', '-Wno-unused-function', '-Wno-unused-label',
             '-Wno-unreachable-code', '-Wno-unused-variable',
-            '-Wno-unused-command-line-argument', '-Wno-unknown-warning-option',
+            '-Wno-unused-command-line-argument', '-Wno-unknown-warning-option', '-g1'
```

Comment:
Can you make the unix debugging level configurable, in case somebody wants the full debug information? For example, this could happen through the `MYPYC_DEBUG_LEVEL` environment variable, with 0, 1 and 2 as possible values (1 being the default).

Reply:
Done!